packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs

```diff
@@ -358,178 +317,151 @@ async fn test_can_trigger_and_index_logdata_event_postgres() {
+    let expected = hex::decode(row.get::<String, usize>(3))
         .unwrap()
         .into_iter()
         .map(|x| x.to_string())
         .collect::<Vec<String>>()
         .join(",");
 
-    assert!((0..=1).contains(&result));
-    assert!(gas_used > 0);
+    assert!((0..=1).contains(&row.get::<BigDecimal, usize>(1).to_u64().unwrap()));
+    assert!(row.get::<BigDecimal, usize>(2).to_u64().unwrap() > 0);
     assert_eq!(expected, "1,1,1,1,1".to_string());
 }
 
 #[actix_web::test]
 #[cfg(all(feature = "e2e", feature = "postgres", feature = "pg-embed-skip"))]
 async fn test_can_trigger_and_index_transferout_event_postgres() {
-    let (fuel_node_handle, test_db, mut srvc) = setup_test_components().await;
-
-    let mut manifest: Manifest =
-        serde_yaml::from_str(assets::FUEL_INDEXER_TEST_MANIFEST).expect("Bad yaml file.");
+    let (node_handle, test_db, mut srvc) = setup_test_components().await;
 
+    let mut manifest = Manifest::try_from(assets::FUEL_INDEXER_TEST_MANIFEST).unwrap();
     update_test_manifest_asset_paths(&mut manifest);
 
-    srvc.register_index_from_manifest(manifest)
-        .await
-        .expect("Failed to initialize indexer.");
+    srvc.register_index_from_manifest(manifest).await.unwrap();
 
     let contract = connect_to_deployed_contract().await.unwrap();
     let app = test::init_service(app(contract)).await;
     let req = test::TestRequest::post().uri("/transferout").to_request();
     let _ = app.call(req).await;
 
     sleep(Duration::from_secs(defaults::INDEXED_EVENT_WAIT)).await;
-    fuel_node_handle.abort();
+    node_handle.abort();
 
     let mut conn = test_db.pool.acquire().await.unwrap();
     let row = sqlx::query("SELECT * FROM fuel_indexer_test_index1.transferout LIMIT 1")
         .fetch_one(&mut conn)
         .await
         .unwrap();
 
-    let recipient: &str = row.get(2);
-    let amount: i64 = row.get(3);
-    let asset_id: &str = row.get(4);
-
     assert_eq!(
-        recipient,
+        row.get::<&str, usize>(2),
         "532ee5fb2cabec472409eb5f9b42b59644edb7bf9943eda9c2e3947305ed5e96"
     );
-    assert_eq!(amount, 1);
-    assert_eq!(asset_id, defaults::TRANSFER_BASE_ASSET_ID);
+    assert_eq!(row.get::<BigDecimal, usize>(3).to_u64().unwrap(), 1);
+    assert_eq!(row.get::<&str, usize>(4), defaults::TRANSFER_BASE_ASSET_ID);
 }
 
-#[actix_web::test]
-#[cfg(all(feature = "e2e", feature = "postgres", feature = "pg-embed-skip"))]
-async fn test_can_trigger_and_index_messageout_event_postgres() {
-    let (fuel_node_handle, test_db, mut srvc) = setup_test_components().await;
-
-    let mut manifest: Manifest =
-        serde_yaml::from_str(assets::FUEL_INDEXER_TEST_MANIFEST).expect("Bad yaml file.");
+// #[actix_web::test]
+// #[cfg(all(feature = "e2e", feature = "postgres", feature = "pg-embed-skip"))]
+// async fn test_can_trigger_and_index_messageout_event_postgres() {
+//     let (node_handle, test_db, mut srvc) = setup_test_components().await;
```

Comment:
Is this test supposed to be commented out? None of the other `pg-embed-skip` tests are commented out, so I'm not sure.

Reply:
@deekerno It's not. I'll address 👌🏽